### PR TITLE
Fix most of the out-of-date font cask installation issues

### PR DESCRIPTION
**Pull request by boltomli**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
No, the casks are legacy and many of them violate naming rule, etc. that I don't touch.
- [ ] The commit message includes the cask’s name and version.
No, casks are many. I can list them in the PR if necessary.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-fonts/pulls
[closed issues]: https://github.com/caskroom/homebrew-fonts/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/font-bungee.rb [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-  sha256 'd5b2a8c80fdab2420bcec03cd4b78451fa10299daea305ff33618d371aa8a62c'
+  version '1.1.0'
+  sha256 'd012a9e6293201c3165feba64342d29c42bb4e67b1cc66e07509c12bab760a6f'
 
   # github.com/djrrb/bungee was verified as official when first introduced to the cask
-  url "https://github.com/djrrb/bungee/releases/download/v#{version}/Bungee-fonts.zip"
+  url "https://github.com/djrrb/Bungee/releases/download/#{version}/Bungee-fonts.zip"
   appcast 'https://github.com/djrrb/bungee/releases.atom',
-          checkpoint: '344b7d19c095589fcf158c7ab4cca696ef4377628aeb14027da0bc61fa40c4d3'
+          checkpoint: '5bf9a5c1e404893bc4b5fa3681f7622b8f4998246ebcf5ffb9a48cb1798ec71e'
```
Comment (miccal) [inline Casks/font-bungee.rb]:
```
 - appcast checkpoint mismatch
Expected: 5bf9a5c1e404893bc4b5fa3681f7622b8f4998246ebcf5ffb9a48cb1798ec71e
Actual: 5f9133d0ed5ba7461c6d3aba4e7d2706951ce25121a8c43ab4bf5f971d1d3f96
```

File: Casks/font-gandom.rb [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
-  version '0.4.4'
-  sha256 'ae2ed95be9915b44c10f8a7d33763cecf29683464ecee4b60fcfcf1516ad3a8f'
+  version '0.4.5'
+  sha256 '4ea2ca208312e405252c8b8373b2a1bab1158f3f5388650a1ac89e195a1b42f0'
 
   # github.com/rastikerdar was verified as official when first introduced to the cask
   url "https://github.com/rastikerdar/gandom-font/releases/download/v#{version}/gandom-font-v#{version}.zip"
   appcast 'https://github.com/rastikerdar/gandom-font/releases.atom',
-          checkpoint: '1c534d1ac2cb060a5b2ac93839fcb411f84fefb65cf0555d805060270280c0ba'
+          checkpoint: '3752706138e6ce47c9c3470ee86ecd6e8841ddec24f00783602bf578f8697a24'
```
Comment (miccal) [inline Casks/font-gandom.rb]:
```
 - appcast checkpoint mismatch
Expected: 3752706138e6ce47c9c3470ee86ecd6e8841ddec24f00783602bf578f8697a24
Actual: ad2a2478a782726dd4f8629cff19c36f83e3dff204ec3aeafc19ce3feaf9876b
```

File: Casks/font-nika.rb [hunk 3]
```diff
@@ -1,12 +1,12 @@
 cask 'font-nika' do
   version '1.0.0'
-  sha256 '0ef9ec73a2406bc81405ea0bbdd81095f0543c6d79c46bc2246f72c7ebf551ba'
+  sha256 'a0e435a23fe971eddb3f39883c606c92b43f9a7b1fc0d762509a291996960d2b'
 
-  url "https://github.com/font-store/font-nika/archive/v#{version}.zip"
+  url "https://github.com/font-store/NikaFont/releases/download/v#{version}/nika.v#{version}.zip"
   appcast 'https://github.com/font-store/font-nika/releases.atom',
-          checkpoint: 'a8141f7742de3164736c5e9b5d6da7d9cc2902e649653f1334d3bcedc70f1dd3'
+          checkpoint: '6a4efdd5658653cc8b20a6fa03350f4225e796efbbea5a96c2f4b4e61cb9327b'
```
Comment (miccal) [inline Casks/font-nika.rb]:
```
 - appcast checkpoint mismatch
Expected: 6a4efdd5658653cc8b20a6fa03350f4225e796efbbea5a96c2f4b4e61cb9327b
Actual: 9c2bba39e9b30c65f708a26a4aa3bec03ef276f0c878115cc6252dd86756e4f1
```

**Comment by boltomli** [comment]
Sorry for the trouble. I was behind some firewall that might be unstable, retried the 3 casks and the new hash is the same with CI now.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**